platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/telemetry/MapboxEventManager.java

```diff
@@ -551,7 +553,16 @@ protected Void doInBackground(Void... voids) {
                     jsonObject.putOpt(MapboxEvent.ATTRIBUTE_SOURCE, evt.get(MapboxEvent.ATTRIBUTE_SOURCE));
                     jsonObject.putOpt(MapboxEvent.ATTRIBUTE_SESSION_ID, evt.get(MapboxEvent.ATTRIBUTE_SESSION_ID));
                     jsonObject.putOpt(MapboxEvent.KEY_LATITUDE, evt.get(MapboxEvent.KEY_LATITUDE));
-                    jsonObject.putOpt(MapboxEvent.KEY_LONGITUDE, evt.get(MapboxEvent.KEY_LONGITUDE));
```

Comment:
@jfirebaugh Ah... my mistake.  I saw the comment attached to Line 554 and thought you were referring to the `jsonObject.putOpt()` for Longitude and didn't follow that you were referring to wrap tests itself.  I follow your thinking now about creating a new `LatLng` with a constant Latitude and the specific Longitude to be wrapped.